Notification refactor

lsemesky opened the pull request:
this is the branch to add to hud and center.

blynch11p commented on app/js/components/notification/NotificationsModal.jsx:
```diff
@@ -0,0 +1,184 @@
+    
+    render: function() { 
+        return (
+            <Modal title="Notifications" ref="modal"
```
This will be `modaltitle="Notifications"` with the recent updates to Modal.jsx.

rkenyon1969 commented:
Changing the following code may resolve "Add" button issue:

`"bookmark_notification_id": notification.id`
to
`"bookmark_notification_id": notification.notificationId`

lsemesky commented:
@rkenyon1969  yes, I have already changed this on the local branch I made for Issue #658. Please feel free to take the ticket yourself if you want to keep working on it but be aware that I have already identified and addressed the issue and have been cleaning up the code before I submit a pr.

rkenyon1969 commented:
No. Please continue to work on this issue. Good job so far. While reviewing this and other PRs came across above solution and thought it might be helpful to your task. - thanks,

blynch11p commented:
:+1: If needed along with #92?

lsemesky commented:
closing because changes are integrated as a part of #92
